src/main/java/hudson/plugins/copyartifact/CopyArtifactPermissionProperty.java

```diff
@@ -53,10 +53,13 @@
 import hudson.model.JobPropertyDescriptor;
 import hudson.model.AbstractProject;
 import hudson.util.FormValidation;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.export.ExportedBean;
 
 /**
  *　Job Property to define projects that can copy artifacts of this project.
  */
+@ExportedBean

```

Comment:
Does not seem to make sense, because it has no exported attributes AFAICT. For example, you could add to `getProjectNameList()`:

```java
@Exported(name = "projects")
```

Maybe better to file that as a separate PR with explanation, and keep this one for the `@Symbol` which should be unrelated?

Reply:
Remove. A bit of cargo culting I thought it helped with @Symbol...